stylegan_code_finder/visualization/segmentation_plotter_lightning.py

```diff
@@ -0,0 +1,68 @@
+import json
+from pathlib import Path
+from typing import List
+import os
+import numpy
+import torch
+from PIL import ImageColor
+from PIL import Image
+import torchvision
+from pytorch_training.images.utils import make_image
+from utils.data_loading import fill_plot_images
+
+
+class SegmentationPlotter:
+    def __init__(self, plot_data_loader, config):
+        assert config['class_to_color_map'] is not None, "Class to Color map must be supplied to SegmentationPlotter!"
+        with Path(config['class_to_color_map']).open() as f:
+            self.class_to_color_map = json.load(f)
+        plot_images = fill_plot_images(plot_data_loader, num_desired_images=config['display_size'])
+        self.label_images = self.label_images_to_color_images(torch.stack(plot_images['segmented']).cuda())

```

Comment:
Remove all these `cuda()` calls after switching to the proper dataloader since lightning will handle this for you